src/main/java/com/cloudbees/jenkins/support/filter/PluginsStopWords.java

```diff
@@ -0,0 +1,25 @@
+package com.cloudbees.jenkins.support.filter;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Set;
+import jenkins.model.Jenkins;
+
+@Extension
+public class PluginsStopWords implements StopWords {
+    @NonNull
+    @Override
+    public Set<String> getWords() {
+        final Set<String> pluginsWords = new HashSet<>();
+        Jenkins.get().pluginManager.getPlugins().forEach(pluginWrapper -> {
```

Comment:
Note that this will include in-house plugins, but those would show up elsewhere anyway.